packages/core/admin/admin/src/components/PluginsInitializer/index.js

```diff
@@ -12,9 +12,9 @@ const PluginsInitializer = () => {
     dispatch({ type: 'SET_PLUGIN_READY', pluginId });
   });
 
-  const hasApluginNotReady = Object.keys(plugins).some(plugin => plugins[plugin].isReady === false);
+  const hasAPluginNotReady = Object.keys(plugins).some(plugin => plugins[plugin].isReady === false);
```

Comment:
I'm not terribly attached to either style, but overall this repository seems to favor this style.